### PR TITLE
Modify readme to show CI status badge.

### PR DESCRIPTION
**Pull request by cijothomas**
This comes after https://github.com/open-telemetry/opentelemetry-dotnet/pull/774 is merged

## Changes
(Please provide a brief description of the changes here.)

### Checklist
- [ ] I ran Unit Tests locally.

For significant contributions please make sure you have completed the following items:

- [ ] Design discussion issue #
- [ ] Changes in public API reviewed

The PR will automatically request reviews from [code owners](https://github.com/open-telemetry/opentelemetry-dotnet/blob/master/CODEOWNERS#L15) and trigger CI build and tests.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # OpenTelemetry .NET
 [![Gitter chat](https://img.shields.io/gitter/room/opentelemetry/opentelemetry-dotnet)](https://gitter.im/open-telemetry/opentelemetry-dotnet)
-[![Build Status](https://dev.azure.com/opentelemetry/pipelines/_apis/build/status/open-telemetry.opentelemetry-dotnet-myget-update?branchName=master)](https://dev.azure.com/opentelemetry/pipelines/_build/latest?definitionId=2&branchName=master)
+
```
Comment (reyang) [inline README.md:3]: Remove this line, otherwise it'll give a line break (the badges will show on two lines instead of one).

File: README.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
 # OpenTelemetry .NET
 [![Gitter chat](https://img.shields.io/gitter/room/opentelemetry/opentelemetry-dotnet)](https://gitter.im/open-telemetry/opentelemetry-dotnet)
-[![Build Status](https://dev.azure.com/opentelemetry/pipelines/_apis/build/status/open-telemetry.opentelemetry-dotnet-myget-update?branchName=master)](https://dev.azure.com/opentelemetry/pipelines/_build/latest?definitionId=2&branchName=master)
+
+[![Build Status](https://github.com/open-telemetry/opentelemetry-dotnet/workflows/.NET%20Linux/badge.svg?branch=master)]
+[![Build Status](https://github.com/open-telemetry/opentelemetry-dotnet/workflows/.NET%20Windows/badge.svg?branch=master)]
+
```
Comment (reyang) [inline README.md:6]: Remove this extra line. Probably run markdownlint.

File: README.md [hunk 3]
```diff
@@ -1,6 +1,9 @@
 # OpenTelemetry .NET
 [![Gitter chat](https://img.shields.io/gitter/room/opentelemetry/opentelemetry-dotnet)](https://gitter.im/open-telemetry/opentelemetry-dotnet)
-[![Build Status](https://dev.azure.com/opentelemetry/pipelines/_apis/build/status/open-telemetry.opentelemetry-dotnet-myget-update?branchName=master)](https://dev.azure.com/opentelemetry/pipelines/_build/latest?definitionId=2&branchName=master)
+
+[![Build Status](https://github.com/open-telemetry/opentelemetry-dotnet/workflows/.NET%20Linux/badge.svg?branch=master)]
+[![Build Status](https://github.com/open-telemetry/opentelemetry-dotnet/workflows/.NET%20Windows/badge.svg?branch=master)]
```
Comment (reyang) [inline README.md:5]: Check the [rendered result](https://github.com/cijothomas/opentelemetry-dotnet/blob/cijothomas/readmebadge/README.md), the brackets seem to be redundant.

image

**Comment by cijothomas** [comment]
> Need to check the render result and fix markdown issues.

done!

File: README.md (hunk 1 above)
Comment (reyang) [reply README.md:3]: Line 3 needs to be removed as well.

Comment (cijothomas) [reply README.md:3]: done

**Comment by cijothomas** [comment]
Final image now:
image

**Pull request merged**